modules/representation_management/spec/lib/representation_management/v0/pdf_constructor/form_2122_spec.rb

```diff
@@ -84,11 +105,31 @@
                                      'representation_management',
                                      'spec',
                                      'fixtures',
-                                     '21-22A',
+                                     '21-22',
                                      'v0',
+                                     'default',
                                      '2122_with_limitations_no_claimant.pdf')
       expect(tempfile.path).to match_pdf_content_of(expected_pdf)
     end
     # The Tempfile is automatically deleted after the block ends
   end
+
+  it 'matches the field values of a pdf with conditions present and no claimant when unflattened' do
```

Comment:
Nit: perhaps reword for readability?

"matches the field values of a pdf with no claimant with conditions present when unflattened"

Reply:
Done!